README.md

```diff
@@ -183,6 +187,8 @@ As a starting point, we recommend carving up your event storm into sub-domains a
 
 ![Sub-domains on an EventStorm](resources/event_storm_sub_domains.png)*Credit: Alberto Brandolini*
 
+More details and templates about context maps can be found in [this repository](https://github.com/ddd-crew/context-mapping)
```

Comment:
I think this link duplicates the link to context mapping above the image. I would simply replace the link above the image to point to this one if its our main reference for context mapping.

If it's just a link to another resource then I would put this link in the tools section below.